internal/configuration/settings/storage.go

```diff
@@ -0,0 +1,50 @@
+package settings
+
+import (
+	"github.com/qdm12/gluetun/internal/constants"
+	"github.com/qdm12/gosettings"
+	"github.com/qdm12/gosettings/reader"
+	"github.com/qdm12/gotree"
+)
+
+// StorageSettings contains settings to configure the storage.
+type StorageSettings struct {
```

Comment:
nit rename to just `Storage`, since we're already in the settings package.

Reply:
There is an existing `Storage` interface in the `settings` package that conflicts. I didn't want to make this PR much larger by trying to tackle that at the same time.

Reply:
Indeed, my bad! Let's rename the `Storage` interface to `FilterChoicesGetter` , you have the 🟢 light! Thanks for the initial thought of keeping the PR smaller, it's appreciated!